packages/babel-plugin-remove-graphql-queries/package.json

```diff
@@ -5,9 +5,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/gatsbyjs/gatsby.git",
-    "directory": "packages/babel-plugin-remove-graphql-queries"
+    "directory": "packages\\babel-plugin-remove-graphql-queries"
   },
-  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/babel-plugin-remove-graphql-queries#readme",
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages\\babel-plugin-remove-graphql-queries#readme",
```

Comment:
Why do we have this here? Basically every affected `package.json` contains similar change and I suspect that's what breaks things at `yarn check-repo-fields`.

Reply:
I think running prettier did it because I haven't touched these manually. Thanks !